### PR TITLE
Add admin-orders

### PR DESCRIPTION
**Pull request by kevinch**
Migrate messages from admin-orders app.

### CODE REVIEW

File: messages/en/vtex.admin-orders@0.x.json [hunk 1]
```diff
@@ -0,0 +1,145 @@
+{
+  "admin.orders-menu-button": "All orders v2",
```
Comment (tlgimenes) [inline messages/en/vtex.admin-orders@0.x.json]: these messages should be scoped by "admin/" so they don't end up in the store

Comment (kevinch) [reply messages/en/vtex.admin-orders@0.x.json]: Updated.

**Review by kaisermann — APPROVED** [review]

**Review by matheusps — APPROVED** [review]

**Review by lucasaarcoverde — APPROVED** [review]

File: messages/en/vtex.admin-orders@0.x.json [hunk 2]
```diff
@@ -0,0 +1,145 @@
+{
```
Comment (estacioneto) [inline messages/en/vtex.admin-orders@0.x.json:1]: Should it have a CHANGELOG? I think you're going to release a version, am I right?

**Comment by kevinch** [comment]
Won't do due to the Crowdin integration.

**Pull request closed without merging**